sdk/trace/provider.go

```diff
@@ -38,7 +38,18 @@ const (
 // TracerProviderConfig
 type TracerProviderConfig struct {
 	processors []SpanProcessor
-	config     Config
+
+	// sampler is the default sampler used when creating new spans.
+	sampler Sampler
+
+	// idGenerator is for internal use only.
```

Comment:
This comment just reiterates the fact this field is unexported.

```suggestion
	// idGenerator is used to generate all Span and Trace IDs when needed.
```